bin/servicerouter.py

```diff
@@ -558,7 +562,7 @@ def get_app(self, appid):
     # Lists all running apps.
     def list(self):
         logger.info('fetching apps')
-        return self.api_req('GET', ['apps'])["apps"]
+        return self.api_req('GET', ['apps', '?embed=apps.tasks'])["apps"]
```

Comment:
To pass in this sort of arguments we want to use kwargs to pass it through to the requests library like http://docs.python-requests.org/en/latest/user/quickstart/#passing-parameters-in-urls

So `return self.api_req('GET', ['apps'], params={'embed': 'apps.tasks'})["apps"]`